models/issue.go

```diff
@@ -246,6 +246,7 @@ func (issue *Issue) APIFormat() *api.Issue {
 
 	apiIssue := &api.Issue{
 		ID:       issue.ID,
+		URL:      issue.HTMLURL(),
```

Comment:
I'm not sure about that. From GitHub's API it seems that it should be the API URL to get the issue, not the HTML URL: https://developer.github.com/v3/issues/#get-a-single-issue

```
"url": "https://api.github.com/repos/octocat/Hello-World/issues/1347",
```